code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1494,7 +1498,7 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 			if(atk_verb == ATTACK_EFFECT_KICK) //kicks never miss (provided your species deals more than 0 damage)
 				miss_chance = 0
 			else
-				miss_chance = min((user.dna.species.punchdamagehigh/user.dna.species.punchdamagelow) + user.getStaminaLoss() + (user.getBruteLoss()*0.5), 100) //old base chance for a miss + various damage. capped at 100 to prevent weirdness in prob()
+				miss_chance = min(clamp(((puncherstam + puncherbrute)*0.5) - ((punchedstam + punchedbrute)*0.5), 0, 100)) //probability of miss based on half your stamina and brute total against half their stamina and brute total. Capped at max 100 and min 0 to prevent weirdness in prob()
```

Comment:
> ((puncherstam + puncherbrute)*0.5) - ((punchedstam + punchedbrute)*0.5)
By this new logic, punches shouldm't miss anymore.

Reply:
I might shrink the bonus down a little more just to avoid completely removing the miss chance. But the intent is that two people fighting each other should probably be able to punch one another unless they're stamdraining themselves sprinting, and if you're trying to punch someone freshfaced it'll start incurring miss chance.

Reply:
Also because of basing miss chance on the division of punchhigh and low, when you have incredibly high variance on your punches, the numbers get...fucky. for instance, mushroom people miss more than humans because their punches had higher variance. As did military synths, corporate agents and some golems. This only got sillier when skooma was involved, with some of these races accuring weird rounding numbers or just flat out missing more. So removing the punchhigh/punchlow from the calc was more or less to keep it relatively sane.

Speaking of skooma had I not changed it originally it would have resulted in skooma users missing with near certainty. Thank fuck for that.

Reply:
I MEANT HIT MORE NOT LESS AAAH IT'S LATE

Golems and fucking tiny miss chance but it still gets weird with skooma. Look my point is that the calculation dumb and so am I.